### ISSUE TITLE
Support for `text-stroke`

### ISSUE THREAD

**Issue by davidsainez** [qq15725/modern-screenshot#17]
Hello, thanks for a great tool.

I wonder if the css property `text-stroke` is meant to be supported? Everything else works great, but text-stroke does not appear. Tested on both Chromium and Firefox (Linux Box). I'm using `modern-screenshot` version 4.4.9.

I would be happy to make a PR if pointed in the right direction. Cheers!

**Comment by qq15725** [comment]
Could you please provide me with the code of the problem? I don't seem to find the problem there

[Playground link](https://sfc.vuejs.org/#eNp9Uk1v2zAM/SuELk2B2C62nbwkwA477DCgGLabLqpN22qsD0h0nCzwfx/9ka5pih4EiOTje4+izuKb9+mhQ5GLTSyC9gQRqfM7abXxLhCcoXTmt3u0NQxQBWfgzrgSg00Yj2hj4+juFTpgtQZnf7rOEpYvPazBKGkLZyNBcI6+t7Ad0av7S7ZUpP78+vE/Le0L0UrFky1gdQ/bHZylhQs6Pai2Q+5RvdLMsZhdzRJzlakGPptsHpGH44DQ+FYRcgSwKfVhVN1KMTdKAUWrYlwSUkwwBkav7O6xVac6sLOSOcfExJExycTNd21qyGMouH8xyowZVzfZK2EOI53a2UM6Cs2zAfS6pCaHzw8P/vh1TjWo64auc5WzlET9F3P4gmZJJj0+7TUlhEcuUnB7Ln/yRx6wfA9S6bZNCte6kEPfaMIJNEwPNrkTazHvNzHKp8/RWf4vk1G5FKIU+cW6FLzsMZaiIfIxz7JYFeMve46pC3XGtzTwVrXBFKNJnoLrIwYmlmJ94bj5ZNeMnfX7Oi2cyW6AvIZImbYlHlNzxcnCGQsdMCQBuR4wfOTzDfTG6/JKgxj+AcVXIOg=)

<img width="1279" alt="image" src="https://user-images.githubusercontent.com/19576382/228853179-02a99eda-481f-43eb-ab3d-03b98cf06ea3.png">

**Comment by davidsainez** [comment]
Hello, I'm sorry for the huge delay.

But I finally got around to do some debugging and thought you might be interested. It turns out that the text-stroke copying is somehow dependent on color! This is probably why we were seeing different output. I hope this is useful. For now, a quick hack I'm using is to use `#000001` as an approximation to black.
 
### Example output

When using `-webkit-text-stroke: 1px #000001;` : 
```html
<div class="view_body" style="transition-property: none; -webkit-text-stroke: 1px rgb(0, 0, 1); -webkit-locale: &quot;en&quot;; width: 120.883px; transform-origin: 60.4414px 28px; text-shadow: rgb(0, 0, 0) 2px 2px 0px; text-emphasis-color: rgb(240, 117, 158); text-decoration: none solid rgb(240, 117, 158); perspective-origin: 60.4375px 28px; outline-color: rgb(240, 117, 158); min-width: auto; min-inline-size: auto; min-height: auto; min-block-size: auto; inline-size: 120.883px; height: 56px; font-size: 56px; font-family: __Luckiest_Guy_71c594, __Luckiest_Guy_Fallback_71c594; column-rule-color: rgb(240, 117, 158); color: rgb(240, 117, 158); caret-color: rgb(240, 117, 158); box-sizing: border-box; border-color: rgb(240, 117, 158); border-inline-color: rgb(240, 117, 158); border-block-color: rgb(240, 117, 158); block-size: 56px;"><div style="font-size: 56px; transition-property: none; -webkit-text-stroke: 1px rgb(0, 0, 1); -webkit-locale: &quot;en&quot;; width: 120.883px; transform-origin: 60.4414px 28px; text-shadow: rgb(0, 0, 0) 2px 2px 0px; text-emphasis-color: rgb(240, 117, 158); text-decoration: none solid rgb(240, 117, 158); perspective-origin: 60.4375px 28px; outline-color: rgb(240, 117, 158); inline-size: 120.883px; height: 56px; font-family: __Luckiest_Guy_71c594, __Luckiest_Guy_Fallback_71c594; column-rule-color: rgb(240, 117, 158); color: rgb(240, 117, 158); caret-color: rgb(240, 117, 158); box-sizing: border-box; border-color: rgb(240, 117, 158); border-inline-color: rgb(240, 117, 158); border-block-color: rgb(240, 117, 158); block-size: 56px;">xxx</div></div>
```

And when I use `-webkit-text-stroke: 1px #000000;`:
```html
 <div class="view_body" style="transition-property: none; -webkit-text-stroke-width: 1px; -webkit-locale: &quot;en&quot;; width: 120.883px; transform-origin: 60.4414px 28px; text-shadow: rgb(0, 0, 0) 2px 2px 0px; text-emphasis-color: rgb(240, 117, 158); text-decoration: none solid rgb(240, 117, 158); perspective-origin: 60.4375px 28px; outline-color: rgb(240, 117, 158); min-width: auto; min-inline-size: auto; min-height: auto; min-block-size: auto; inline-size: 120.883px; height: 56px; font-size: 56px; font-family: __Luckiest_Guy_71c594, __Luckiest_Guy_Fallback_71c594; column-rule-color: rgb(240, 117, 158); color: rgb(240, 117, 158); caret-color: rgb(240, 117, 158); box-sizing: border-box; border-color: rgb(240, 117, 158); border-inline-color: rgb(240, 117, 158); border-block-color: rgb(240, 117, 158); block-size: 56px;"><div style="font-size: 56px; transition-property: none; -webkit-text-stroke-width: 1px; -webkit-locale: &quot;en&quot;; width: 120.883px; transform-origin: 60.4414px 28px; text-shadow: rgb(0, 0, 0) 2px 2px 0px; text-emphasis-color: rgb(240, 117, 158); text-decoration: none solid rgb(240, 117, 158); perspective-origin: 60.4375px 28px; outline-color: rgb(240, 117, 158); inline-size: 120.883px; height: 56px; font-family: __Luckiest_Guy_71c594, __Luckiest_Guy_Fallback_71c594; column-rule-color: rgb(240, 117, 158); color: rgb(240, 117, 158); caret-color: rgb(240, 117, 158); box-sizing: border-box; border-color: rgb(240, 117, 158); border-inline-color: rgb(240, 117, 158); border-block-color: rgb(240, 117, 158); block-size: 56px;">xxx</div></div>
```

**Comment by davidsainez** [comment]
Also, forgot to mention, it occurs if the color is set with `-webkit-text-stroke: 1px black;` and even when separating the property into `-webkit-text-stroke-color` and `-webkit-text-stroke-width`.

**Comment by davidsainez** [comment]
I've been poking around the code a bit, and I suspect that this bug occurs when filtering out the default styles. This would make sense because only the `#000000` value (which I assume would be the default) gets dropped.

**Comment by qq15725** [comment]
Fixed in 4.4.17
